src/config.rs

```diff
@@ -70,6 +72,8 @@ struct InputText {
     margin: Margin,
     #[def = "Padding::from_pair(1.7, -4.0)"]
     padding: Padding,
+    #[def = "Radius::all(f32::INFINITY)"]
```

Comment:
It's not really an infinity, isn't it?

Reply:
You're right, `f32::MAX` is probably more appropriate.